stdlib/exn.mli

```diff
@@ -0,0 +1,100 @@
+(**************************************************************************)
+(*                                                                        *)
+(*                                 OCaml                                  *)
+(*                                                                        *)
+(*             Xavier Leroy, projet Cristal, INRIA Rocquencourt           *)
+(*                                                                        *)
+(*   Copyright 2018 Institut National de Recherche en Informatique et     *)
+(*     en Automatique.                                                    *)
+(*                                                                        *)
+(*   All rights reserved.  This file is distributed under the terms of    *)
+(*   the GNU Lesser General Public License version 2.1, with the          *)
+(*   special exception on linking described in the file LICENSE.          *)
+(*                                                                        *)
+(**************************************************************************)
+
+(** Exception values.
+
+    Facilities for printing exceptions and obtaining traces.
+
+    @since 4.08 *)
+
+type t = exn = ..
+(** The type for exceptions. *)
+
+val to_string: t -> string
+(** [to_string e] returns a string representation of the exception [e]. *)
+
```

Comment:
I think a heading `{1:tracing Tracing}` could be introduced for functions related to tracing.

Reply:
Thanks, done.